mmpose/core/evaluation/mesh_eval.py

```diff
@@ -0,0 +1,62 @@
+import numpy as np
+
+
+def compute_similarity_transform(S1, S2):
```

Comment:
is `S` short for Surface?

Reply:
I adapt the codes from SPIN directly.  In SPIN, 'S' is used to represent 3D points and I don't know what it is short for. 
If it's not clear enough, I can change the name later.

Reply:
need to add licence header if the file is adapted from other repo. see examples [here](https://github.com/open-mmlab/mmpose/blob/7672061b28b9b67482852f8f6f3ac8425026a456/mmpose/core/post_processing/nms.py)

yeah better to change the name to something meaningful